src/backend/parser/analyze.c

```diff
@@ -1413,6 +1416,8 @@ transformSelectStmt(ParseState *pstate, SelectStmt *stmt)
 	/* process the FROM clause */
 	transformFromClause(pstate, stmt->fromClause);
 
+	if(sql_dialect == SQL_DIALECT_TSQL && bbf_rewrite_targetlist_hook)
+		(*bbf_rewrite_targetlist_hook)(stmt->targetList);
```

Comment:
All the query would go through this function and makes more risk for other queries other than select into, would you add a condition in here only allow select into to enter this rewrite check ?

Reply:
Removed this hook as it is no longer needed. order by works fine without moving identity function at the end of target list. I used tleSortGroupRef  which is an Index and remains same even when we remove identity function from target list in transformSelectIntoStmt